build.js

```diff
@@ -618,16 +619,91 @@ module.exports = function (gulpWrapper, ctx) {
         gulpWrapper.seq(["__internal-build"], callback);
     });
 
+    /**
+     * Build Documentation Database(s) for Search task
+     */
+    gulp.task('__build-db', function (callback) {             
+        if (ctx.type === "documentation") {
```

Comment:
This "if" should be done on __internal_build instead. It will spare jumping into another task for most packages and leaving a cleaner log. Can you change this plz?